goss-testing/tests/ncn/goss-k8s-metallb_configmap-status.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+{{ $scripts := .Env.GOSS_BASE | printf "%s/scripts" }}
+{{ $logrun := $scripts | printf "%s/log_run.sh" }}
+command:
+    {{ $testlabel := "check_k8s-metallb-status" }}
+    {{$testlabel}}:
+        title: Verify that Metal Load Balancer for Kubernetes is running on service ingress
+        meta:
+            desc: If this test fails, then verify that load balance is properly configured on the ingress service. See troubleshooting/known_issues/check_system_management_monitoring_tools.md in the CSM documentation for additional remediation steps.
+            sev: 0
+        exec: |-
+            "{{$logrun}}" -l "{{$testlabel}}" \
+                kubectl -n istio-system describe service istio-ingressgateway-cmn
+        stdout:
+          - "Annotations"
```

Comment:
I'm curious why this is one of the test criteria. I'm not saying it's incorrect, it's just not clear why it is what we're checking. I recommend adding a comment explaining this criteria, so that future people looking at the test will understand why that is being checked.